llvm/include/llvm/TargetParser/RISCVTargetParser.h

```diff
@@ -24,6 +24,14 @@ class Triple;
 
 namespace RISCV {
 
+namespace RISCVExtensionBitmaskTable {
+struct RISCVExtensionBitmask {
+  const char *Name;
+  unsigned GroupID;
```

Comment:
Can we store just a single index value where 0-63 means its in group 0, 64-127 means it is in group 1, 128-191 means its in group 2, etc.

Reply:
Can we use the bit position in the group instead of the mask here? It will make the structure more compact. There is currently a 4 byte gap between the group and the bit mask.

Reply:
Updated